mlflow/tracking/_tracking_service/utils.py

```diff
@@ -113,7 +114,9 @@ def _get_sqlalchemy_store(store_uri, artifact_uri):
     from mlflow.store.tracking.sqlalchemy_store import SqlAlchemyStore
 
     if artifact_uri is None:
-        artifact_uri = DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH
+        artifact_uri = (
+            os.environ.get(_ARTIFACT_ROOT_ENV_VAR, None) or DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH
```

Comment:
Is there a reason you preferred this over
```suggestion
            os.environ.get(_ARTIFACT_ROOT_ENV_VAR, DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH)
```
While I understand an empty/unset `MLFLOW_ARTIFACT_ROOT`  would be replaced by the default value, is there another reason?

From the code above (`get_tracking_uri`), it appears that if the `_TRACKING_URI_ENV_VAR` environment variable is set but empty, we would still use its empty value. Thus the two behavior are different which might not be what we want.

Reply:
No reason indeed, agree !